uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/Upgrader/Step/1022.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+trait CRM_HRLeaveAndAbsences_Upgrader_Step_1022 {
+  
+  /**
+   * Default existing work pattern to As per contract (new default)
+   *
+   * @return bool
+   */
+  public function upgrade_1022()
+  {
+    civicrm_api3('ContactWorkPattern', 'get', [
+      'return' => ['id'],
+      'api.ContactWorkPattern.update' => [
+        'change_reason' => 1,
```

Comment:
Instead of hardcoding the change reason can you fetch the `value` of the OptionValue based on machine name and `option_group_id`. It's best not to depend on `value` as we've had problems before where it has been different depending on the system.